.github/workflows/cd.yaml

```diff
@@ -126,14 +136,16 @@ jobs:
           # Trim `v` prefix if exists
           CHART_VERSION=${CHART_VERSION#v}
           IMAGE_VERSION=${IMAGE_VERSION#v}
+          CHART_VERSION=${CHART_VERSION:-0.0.0-${IMAGE_VERSION}}
 
           # Build/package the chart
           echo "Setting chart version to ${CHART_VERSION} and image version to ${IMAGE_VERSION}"
           helm package . --version "${CHART_VERSION}" --app-version "${IMAGE_VERSION}"
           ARTIFACT_NAME=$(find . -name '*.tgz' -exec basename {} \; | head -n 1)
 
           # Publish the chart
-          helm push "$ARTIFACT_NAME" "oci://$HELM_PRIVATE_ECR_REPO"
+          # helm push "$ARTIFACT_NAME" "oci://${{ vars.HELM_PRIVATE_ECR_REPO }}"
+          helm push "$ARTIFACT_NAME" "oci://603330915152.dkr.ecr.us-west-2.amazonaws.com/gravitational/charts/"
```

Comment:
Confirmed that build works this way.
Now need to wait to get environment variables updated in github before reverting last commit and merging this.